CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.16.0-rc.0 / 2020-01-30
+
+* [CHANGE] PromQL: Remove MaxConcurrent from the PromQL engine opts #6712
+* [FEATURE] PromQL: Support trailing commas in grouping opts #6480
+* [FEATURE] PromQL: add absent_over_time query function #6490
+* [FEATURE] Adding optional logging of queries to their own file #6520
+* [ENHANCEMENT] Remote write: Improve sharding calculation in cases where we would always be consistently behind by tracking pendingSamples ##6511
+* [ENHANCEMENT] React UI: Add support for rules page and "Xs ago" duration displays #6503
+* [ENHANCEMENT] promtool: Add support for line-column numbers for invalid rules output #6533
+* [ENHANCEMENT] ReactUI: alerts page, replace filtering togglers tabs with checkboxes #6543
+* [ENHANCEMENT] PromQL: Use PositionRange in parser errors to improve error messages #6634
```

Comment:
Might make sense to just write that the PromQL parser has been (almost) completely rewritten, which results in better error messages.

The use of `positionRange` is just an implementation detail, that doesn't need to go into the change log.

Reply:
sure, are there any other PR's that should be listed under that message as well?

Reply:
The parser rewrite was spread over roughly 20 PRs. I don't think it makes sense to list them all.

I guess  #6548 was the most important one.